config/packages/db_tools.yaml

```diff
@@ -44,3 +44,22 @@ db_tools:
         #yaml:
             #- connection_one: '%kernel.project_dir%/config/anonymizations/connection_one.yaml'
             #- connection_two: '%kernel.project_dir%/config/anonymizations/connection_two.yaml'
+
+    # Connection-specific configuration.
+    # Keys are DBAL connection names registered in container configurfation.
+    #connections:
+        #default:
```

Comment:
I find a bit odd this key `connections`, I'd rather see a dedicated section like `storage` as first level key